packages/epo-widget-lib/src/charts/ForeignObject/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { FunctionComponent, PropsWithChildren } from "react";
+
+const ForeignObject: FunctionComponent<
+  PropsWithChildren<{ className?: string }>
+> = ({ children, className }) => {
+  return (
+    <foreignObject
```

Comment:
extract out the `ForeignObject` pattern I have been using for tooltips and other non-SVG chart content